lib/modules/manager/npm/post-update/types.ts

```diff
@@ -30,9 +30,16 @@ export interface GenerateLockFileResult {
   stdout?: string;
 }
 
+// the dependencies schema is different for v6 and other lockfile versions
```

Comment:
Seems like this should be in a common types file instead of post-update, but perhaps it can be changed later